src/aws-adapter.js

```diff
@@ -47,22 +47,35 @@ function eventToQueryString(event) {
  * headers, separately.
  *
  * @param raw raw headers object
- * @returns object containing a property 'headers' and a property 'multiValueHeaders'
+ * @param log log
+ * @returns object containing a property 'headers' and a property 'cookies'
  */
-function splitHeaders(raw) {
+function splitHeaders(raw, log) {
   const headers = {};
-  const multiValueHeaders = {};
+  const cookies = [];
 
   Object.entries(raw).forEach(([name, value]) => {
     if (Array.isArray(value)) {
-      multiValueHeaders[name] = value;
+      const lwrname = name.toLowerCase();
+      switch (lwrname) {
+        case 'vary':
+          headers[name] = value.join(', ');
```

Comment:
I wouldn't add an extra space
```suggestion
          headers[name] = value.join(',');
```

Reply:
why not? the same response in CF incidentally has exactly the format:
```
Vary: x-forwarded-host, Accept-Encoding
```
I'm quite confident that adding that space does not break anything in between